[CI] SearchableSnapshotsCanMatchOnCoordinatorIntegTests testSearchableSnapshotShardsAreSkippedBySearchRequestWithoutQueryingAnyNodeWhenTheyAreOutsideOfTheQueryRange failing

**Build Scans:**
- [elasticsearch-intake #12705 / part2](https://gradle-enterprise.elastic.co/s/rjtwka43temuu)
- [elasticsearch-periodic-platform-support #4797 / opensuse-leap-15_platform-support-unix](https://gradle-enterprise.elastic.co/s/pglt6hpypa46m)
- [elasticsearch-periodic-platform-support #4797 / sles-12_platform-support-unix](https://gradle-enterprise.elastic.co/s/4ymonko764gvk)
- [elasticsearch-periodic-platform-support #4797 / oraclelinux-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/mc7zaklh7n374)
- [elasticsearch-periodic-platform-support #4797 / sles-15_platform-support-unix](https://gradle-enterprise.elastic.co/s/bxscfofiuco6w)
- [elasticsearch-periodic-platform-support #4797 / windows-2016_checkpart2_platform-support-windows](https://gradle-enterprise.elastic.co/s/6xbyfj7esksyg)
- [elasticsearch-periodic #4820 / release-tests](https://gradle-enterprise.elastic.co/s/idvumkxsduixa)
- [elasticsearch-periodic-platform-support #4792 / rocky-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/ucmffbju7ylqy)
- [elasticsearch-periodic #4820 / openjdk17_checkpart2_java-matrix](https://gradle-enterprise.elastic.co/s/g6hrl2titgwlm)
- [elasticsearch-periodic #4820 / openjdk21_checkpart2_java-matrix](https://gradle-enterprise.elastic.co/s/nzjadbiflolu4)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:searchable-snapshots:internalClusterTest" --tests "org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests.testSearchableSnapshotShardsAreSkippedBySearchRequestWithoutQueryingAnyNodeWhenTheyAreOutsideOfTheQueryRange" -Dtests.seed=D417436B8B80F0F -Dtests.locale=bn-Beng-BD -Dtests.timezone=UTC -Druntime.java=23
```

**Applicable branches:**
8.16

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testSearchableSnapshotShardsAreSkippedBySearchRequestWithoutQueryingAnyNodeWhenTheyAreOutsideOfTheQueryRange),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: no shard should be marked as skipped
```

**Issue Reasons:**
- [8.16] 5 consecutive failures in test testSearchableSnapshotShardsAreSkippedBySearchRequestWithoutQueryingAnyNodeWhenTheyAreOutsideOfTheQueryRange
- [8.16] 20 failures in test testSearchableSnapshotShardsAreSkippedBySearchRequestWithoutQueryingAnyNodeWhenTheyAreOutsideOfTheQueryRange (18.5% fail rate in 108 executions)
- [8.16] 2 failures in step openjdk17_checkpart2_java-matrix (50.0% fail rate in 4 executions)
- [8.16] 2 failures in step rocky-8_platform-support-unix (66.7% fail rate in 3 executions)
- [8.16] 3 failures in pipeline elasticsearch-periodic-platform-support (75.0% fail rate in 4 executions)
- [8.16] 2 failures in pipeline elasticsearch-periodic (50.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -132,9 +132,6 @@ tests:
 - class: org.elasticsearch.reservedstate.service.FileSettingsServiceTests
   method: testInvalidJSON
   issue: https://github.com/elastic/elasticsearch/issues/116521
-- class: org.elasticsearch.xpack.searchablesnapshots.SearchableSnapshotsCanMatchOnCoordinatorIntegTests
-  method: testSearchableSnapshotShardsAreSkippedBySearchRequestWithoutQueryingAnyNodeWhenTheyAreOutsideOfTheQueryRange
-  issue: https://github.com/elastic/elasticsearch/issues/116523
 - class: org.elasticsearch.reservedstate.service.RepositoriesFileSettingsIT
   method: testSettingsApplied
   issue: https://github.com/elastic/elasticsearch/issues/116694

--- a/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java
+++ b/x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java
@@ -379,7 +379,7 @@ public class SearchableSnapshotsCanMatchOnCoordinatorIntegTests extends BaseFroz
                     }
                     if (searchShardsResponse != null) {
                         for (SearchShardsGroup group : searchShardsResponse.getGroups()) {
-                            assertFalse("no shard should be marked as skipped", group.skipped());
+                            assertTrue("the shard is skipped because index value is outside the query time range", group.skipped());
                         }
                     }
                 }

